### ISSUE TITLE
json skip serialize default primitive values

### ISSUE THREAD

**Issue by avrecko** [odnoklassniki/one-nio#68]
I am looking at the json serialization. A bit surprised I am not finding an option to skip serialization of default values. Is this by design?

**Comment by incubos** [comment]
> I am looking at the json serialization. A bit surprised I am not finding an option to skip serialization of default values. Is this by design?

Yep, as [mentioned in JavaDoc](https://github.com/odnoklassniki/one-nio/blob/master/src/one/nio/serial/JsonReader.java#L31):
```java
/**
 * Beware: this is NOT a complete and fully compliant JSON parser!
 * Its main purpose is to decode typical simple queries without third-party dependencies.
 */
```

But contributions are always welcome!
